CMakeLists.txt

```diff
@@ -23,7 +23,7 @@
 
 cmake_minimum_required (VERSION 3.11 FATAL_ERROR)
 
-find_package(IRODS 4.2.11 EXACT REQUIRED CONFIG)
+find_package(IRODS 4.3.0 EXACT REQUIRED CONFIG)
 set(IRODS_CLIENT_REVISION "1")
```

Comment:
Should we knock this back down to 0?

Reply:
indeed.

Reply:
diff still says 1... maybe you just haven't pushed yet?

Reply:
Not sure what happened but this is done now.